web/test/pages/profile/profile-starting.test.tsx

```diff
@@ -1354,6 +1378,299 @@ describe("profile - starting business", () => {
+
+      clickSave();
+      const profileAlert = screen.getByTestId("profile-error-alert");
+
+      await waitFor(() => {
+        expect(profileAlert).toBeInTheDocument();
+      });
+      expect(within(profileAlert).getByText(Config.formation.fields.addressLine1.label)).toBeInTheDocument();
+      expect(within(profileAlert).getByText(Config.formation.fields.addressLine2.label)).toBeInTheDocument();
+      expect(
+        within(profileAlert).getByText(Config.formation.fields.addressMunicipality.label)
+      ).toBeInTheDocument();
+      expect(
+        within(profileAlert).getByText(Config.formation.fields.addressZipCode.label)
+      ).toBeInTheDocument();
+    });
+
+    it("renders disabled profile address fields when business has formed", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          businessPersona: "STARTING",
+        }),
+        formationData: generateFormationData({
+          completedFilingPayment: true,
+          formationFormData: generateFormationFormData({
+            addressLine1: "123 Testing Road",
+            addressLine2: "",
+            addressMunicipality: generateMunicipality({ displayName: "Allendale" }),
+            addressState: {
+              name: "New Jersey",
+              shortCode: "NJ",
+            },
+            addressZipCode: "07781",
+          }),
+        }),
+      });
+      renderPage({ business });
+
+      expect(screen.getByLabelText("Address line1")).toBeDisabled();
+      expect(screen.getByLabelText("Address line2")).toBeDisabled();
+      expect(screen.getByLabelText("Address municipality")).toBeDisabled();
+      expect(screen.getByLabelText("Address zip code")).toBeDisabled();
+    });
+
+    it("does not render disabled profile address fields when business has not been formed", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          businessPersona: "STARTING",
+        }),
+        formationData: generateFormationData({
+          completedFilingPayment: false,
+          formationFormData: generateFormationFormData({
+            addressLine1: "123 Testing Road",
+            addressLine2: "",
+            addressMunicipality: generateMunicipality({ displayName: "Allendale" }),
+            addressState: {
+              name: "New Jersey",
+              shortCode: "NJ",
+            },
+            addressZipCode: "07781",
+          }),
+        }),
+      });
+      renderPage({ business });
+
+      expect(screen.getByLabelText("Address line1")).toBeEnabled();
+      expect(screen.getByLabelText("Address line2")).toBeEnabled();
+      expect(screen.getByLabelText("Address municipality")).toBeEnabled();
+      expect(screen.getByLabelText("Address zip code")).toBeEnabled();
+    });
+  });
+
   const getIndustryValue = (): string => {
     return (screen.queryByTestId("industryId") as HTMLInputElement)?.value;
```

Comment:
[pebble] I think maybe we should have a test confirming that this doesn't appear for Foreign

Reply:
I'm not entirely sure I saw logic like that anywhere but its possible I've missed it

Reply:
💡 - added